public/css/form.less

```diff
@@ -84,6 +85,15 @@
   }
 }
 
+.timezone-picker {
+  display: inline;
```

Comment:
Apply CSS classes `icinga-form, inline, icinga-controls` to the form and this is not required. The extra div around the select is also not required this way.